akka-typed/src/main/scala/akka/typed/Deployment.scala

```diff
@@ -156,17 +169,23 @@ case object EmptyProps extends Props {
  *
  * The default configuration if none of these options are present is to run
  * the actor on the same executor as its parent.
+ *
+ * INTERNAL API
  */
-sealed trait DispatcherSelector extends Props
+@DoNotInherit
```

Comment:
Internal implies "do not touch at all, also dont sub class" I'd just keep that one